build/images/base/build.sh

```diff
@@ -129,29 +135,73 @@ if $PULL; then
     done
 fi
 
-docker build $PLATFORM_ARG --target cni-binaries \
-       --cache-from antrea/cni-binaries:$CNI_BINARIES_VERSION \
-       -t antrea/cni-binaries:$CNI_BINARIES_VERSION \
-       --build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION \
-       --build-arg BUILD_TAG=$BUILD_TAG .
-
-if [ "$DISTRO" == "ubuntu" ]; then
-    docker build $PLATFORM_ARG \
-           --cache-from antrea/cni-binaries:$CNI_BINARIES_VERSION \
-           --cache-from antrea/base-ubuntu:$BUILD_TAG \
-           -t antrea/base-ubuntu:$BUILD_TAG \
-           --build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION \
-           --build-arg SURICATA_VERSION=$SURICATA_VERSION \
-           --build-arg BUILD_TAG=$BUILD_TAG .
-elif [ "$DISTRO" == "ubi" ]; then
-    docker build $PLATFORM_ARG \
-           --cache-from antrea/cni-binaries:$CNI_BINARIES_VERSION \
-           --cache-from antrea/base-ubi:$BUILD_TAG \
-           -t antrea/base-ubi:$BUILD_TAG \
-           -f Dockerfile.ubi \
-           --build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION \
-           --build-arg SURICATA_VERSION=$SURICATA_VERSION \
-           --build-arg BUILD_TAG=$BUILD_TAG .
+if [ "$TYPE" == "unified" ]; then
```

Comment:
Instead of adding the argument, can we just build all required images given a distro?

Reply:
sure